### PR TITLE
FIX #5442 list post impact print function

### LINKED ISSUE #5442: Bug: Date filter on customer invoices list not kept when printing central zone
# Bug

["Date filter on customer invoices list not kept when printing central zone*]
## Environment
- **Version**: [_3.9.2_]
- **OS**: [_Synology last DSM or Ubuntu_]
- **Web server**: [_Apache 2_]
- **PHP**: [_5.6 and 7.0_]
- **Database**: [_MariaDB 5.5.49-0065 et MySQL 5.7.12_]
## Expected and actual behavior

[_When i put a date filter on my list of customer invoices and apply it, it shows only invoices of the month selected. If i open the printer view, i expect to see the list with the filter applied without the window decoration in order to print the list i want... It doesn't work and the list is full without any filter applied._]

### PR DESCRIPTION
**Pull request by ghost**
form get instead of post

### CODE REVIEW

File: htdocs/compta/facture/list.php [hunk 1]
```diff
@@ -589,7 +589,7 @@
 	$massactionbutton=$form->selectMassAction('', $massaction ? array() : array('presend'=>$langs->trans("SendByMail")));
     
     $i = 0;
-    print '<form method="POST" name="searchFormList" action="'.$_SERVER["PHP_SELF"].'">'."\n";
+    print '<form method="GET" name="searchFormList" action="'.$_SERVER["PHP_SELF"].'">'."\n";
```
Comment (eldy) [inline htdocs/compta/facture/list.php:592]: All form must use the POST metho (GET for form is an old behaviour that should be abandonned due to problem, like limits in size by browser or web server, different antispam filtering, url encoding twice, ...)
Fix of bug must be addressed differently.

Comment (ghost) [inline htdocs/compta/facture/list.php:592]: Hi, cause of print button. It use the QUERY_STRING to compose it, i do another PR (that i just see you merge) on develop to manage POST in
